### DISCUSSION TITLE
Upgrade info

### DISCUSSION THREAD

**Discussion by astar24e** [globaleaks/globaleaks-whistleblowing-software#4056] [Q&A]
### Proposal

after updating via script the logos are:
 -f install.log
Reading package lists...
Building dependency tree...
Reading state information...
globalaks is already the newest version (4.15.3).
The following packages were automatically installed and are no longer required:
  libflashrom1 libftdi1-2
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 65 not upgraded.

doing apt-get update it appears to me

Reading package lists... Done
W: http://deb.globaleaks.org/jammy/Release.gpg: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.

it's correct ?

### Motivation and context

correct update

**Comment by evilaliv3** [comment]
Hello @astar24e, thank you for your question

to update to the current safer configuration you could do:

```
rm /etc/apt/trusted.gpg
wget -qO- https://deb.globaleaks.org/globaleaks.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/globaleaks.gpg
echo "deb [signed-by=/etc/apt/trusted.gpg.d/globaleaks.gpg] http://deb.globaleaks.org $DISTRO_CODENAME/" > /etc/apt/sources.list.d/globaleaks.list
apt-get update
```
